src/app/components/LinkButton/LinkButton.tsx

```diff
@@ -3,19 +3,22 @@ import Link, { LinkProps } from 'next/link';
 
 const BUTTON_STYLES = {
   primary: {
-    button: 'usa-button bg-[#224a58] hover:bg-green',
-    text: 'text-white',
+    button:
+      'text-white usa-button bg-violet-warm-60 hover:bg-violet-warm-50 active:bg-violet-warm-70',
   },
   secondary: {
     button:
-      'usa-button hover:bg-red items-center gap-2.5 rounded border-2 border-[#224a58] bg-white px-5 py-3 hover:border-2',
-    text: 'text-[#224a58]',
+      'usa-button !text-violet-warm-60 items-center gap-2.5 rounded border-2 border-violet-warm-60 hover:border-violet-warm-50 bg-white px-5 py-3 hover:border-violet-warm-50 hover:border-2 hover:bg-white active:border-violet-warm-70 hover:!text-violet-warm-50',
+  },
+  transparent: {
+    button:
+      'usa-button items-center gap-2.5 rounded border-2 border-white bg-transparent px-5 py-3 hover:border-2 hover:bg-transparent active:border-violet-warm-70 hover:!text-violet-warm-50',
```

Comment:
This is the `Contact us` button we were chatting about @nickbristow. It has some placeholder styling you can modify once we merge this PR.